python_modules/dagster/dagster_tests/storage_tests/utils/partition_status_cache.py

```diff
@@ -35,6 +38,14 @@
 from dagster_tests.storage_tests.utils.event_log_storage import create_and_delete_test_runs
 
 
+def get_asset_status_cache_value(
```

Comment:
!important

The reason that we're using this instead of get_and_update_asset_status_cache_value is because this is more accurate to how the cache is actually interacted with. In particular, making this change cause the internal build to fail (as expected), because the GQL interface hasn't (yet) been updated to return data for the fields that we're now setting to None.

You don't detect that issue if we build the cache locally using get_and_update_asset_status_cache_value